BUILD.bazel

```diff
@@ -1,18 +1,14 @@
-load("@rules_pkg//:pkg.bzl", "pkg_tar")
+load("@build_bazel_rules_nodejs//:index.bzl", "pkg_npm")
 
 # This declares the release artifact published to npm
-pkg_tar(
+pkg_npm(
```

Comment:
I'm curious if we'll actually need this. It would be simpler just to `npm publish` and use the npm-provided facility to choose which of our source files go into the package.

Reply:
now it is a simple npm pack that does not rely on pkg_npm.